nova/virt/vmwareapi/vmops.py

```diff
@@ -295,14 +304,19 @@ def _get_folder_name(self, name, id):
         # We cannot truncate the 'id' as this is unique across OpenStack.
         return '%s (%s)' % (name[:40], id[:36])
 
-    def _get_project_folder(self, dc_info, project_id=None, type_=None):
```

Comment:
Hi, could you keep the signature the same?
It shouldn't hurt, if it stays the same, but a change might have unintended side effects

Reply:
It is the same. _get_project_folder_path is a separate new method. _get_project_folder signature remains exactly the same (see line 313 of the new version of the file).

Reply:
Sorry, I missed that. Then I'll merge it.